### PR TITLE
Drop travis, add badges and links to README

### PR DESCRIPTION
**Pull request by rbikar**
We have already moved to gh actions for CI, so let's drop
travis now.

This changes also adds badge from tox-tests and codevoc + adds
badges to source, docs and pypi.

This PR move push-docs gh action to a separete file, so it can be 
used by as a status badge.

### CODE REVIEW

**Comment by rbikar** [comment]
run tests

**Comment by rbikar** [comment]
cc @daminijain23

File: README.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 # **ubi-population-tool**
-[![Build Status](https://travis-ci.org/release-engineering/ubi-population-tool.svg?branch=master)](https://travis-ci.org/release-engineering/ubi-population-tool)
-[![Coverage Status](https://coveralls.io/repos/github/release-engineering/ubi-population-tool/badge.svg?branch=master)](https://coveralls.io/github/release-engineering/ubi-population-tool?branch=master)
+[![Build Status](https://github.com/release-engineering/ubi-population-tool/actions/workflows/tox-test.yml/badge.svg)](https://github.com/release-engineering/ubi-population-tool/actions/workflows/tox-test.yml)
+[![codecov](https://codecov.io/gh/release-engineering/ubi-population-tool/branch/master/graph/badge.svg?token=APniN2wa2U)](https://codecov.io/gh/release-engineering/ubi-population-tool/)
 
+- [Source](https://github.com/release-engineering/ubi-population-tool/)
+- [Documentation](https://release-engineering.github.io/ubi-population-tool/)
+- [PyPI](https://pypi.org/project/ubi-population-tool/)
```
Comment (rohanpm) [inline README.md:7]: This is consistent with various other projects but I've been thinking about trying to replace these with badges as well... might look nicer?

**Review by rohanpm — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (rbikar) [reply README.md:7]: ..yeah, why not? Let's try it here :)

**Comment by rbikar** [comment]
run tests

**Comment by rbikar** [comment]
run tests

**Review by rohanpm — APPROVED** [review]

File: .github/workflows/docs.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install Tox
+        run: pip install tox
+      - name: Run Tox
+        run: |
+          tox -e docs
+          scripts/push-docs
```
[the remark this answers is not in the document]
Comment (rbikar) [reply .github/workflows/docs.yml:24]: thx for info, I'll have a look at it.

**Pull request merged**